nginx/nginx.conf

```diff
@@ -0,0 +1,31 @@
+events {
+}
+
+http {
+    server_tokens off;
+    charset utf-8;
+        upstream frontend {
```

Comment:
doesn't look formatted 😅

Reply:
i'm on it

Reply:
@balakrishna-deriv i checked and it seems there's no problem on it.
can you point to where do you see problem?

Reply:
`upstream frontend` is intended here and it doesn't need the indentation